src/Python/qsharp-core/qsharp/clients/iqsharp.py

```diff
@@ -157,18 +165,46 @@ def capture(msg):
                 data = unmap_tuples(json.loads(msg["content"]["data"]["application/json"]))
                 for component, version in data["rows"]:
                     versions[component] = LooseVersion(version)
-        self._execute("%version", output_hook=capture, **kwargs)
+        self._execute("%version", output_hook=capture, _quiet_=True, **kwargs)
         return versions
 
     ## Internal-Use Methods ##
 
-    def _execute_magic(self, magic : str, raise_on_stderr : bool = False, **params) -> Any:
-        return self._execute(f'%{magic} {json.dumps(map_tuples(params))}', raise_on_stderr=raise_on_stderr)
+    def _execute_magic(self, magic : str, raise_on_stderr : bool = False, _quiet_ : bool = False, **kwargs) -> Any:
+        return self._execute(
+            f'%{magic} {json.dumps(map_tuples(kwargs))}',
+            raise_on_stderr=raise_on_stderr, _quiet_=_quiet_,
+            **kwargs
```

Comment:
Perhaps `**kwargs` shouldn't be passed through here, since it's being converted to JSON as an argument to the magic command? I noticed the CI failure and I assume it's related to this.

Reply:
Good catch, thank you! I must have made that chance accidentally renaming things to be consistent, will revert that line.